docs/user/spark/sparksql.rst

```diff
@@ -109,11 +109,11 @@ feature type, do the following:
     val dataFrame = sparkSession.read
       .format("geomesa")
       .options(dsParams)
-      .option("geomesa.feature", typeName)
+      .option(GEOMESA_SQL_FEATURE, typeName)
```

Comment:
we should include the import here if we're going to go with this controversial decision ;)